alembic/versions/06b9ad98e471_added_second_shirt_column.py

```diff
@@ -0,0 +1,45 @@
+"""Added second_shirt column to the Attendee table
+
+Revision ID: 06b9ad98e471
+Revises: 4947b38a18b1
```

Comment:
FYI, there is a bug in the `sep alembic` code that automatically determines the `down_revision`, see https://github.com/magfest/ubersystem/issues/2911. (If you want to be a hero, you can try fixing the sep alembic bug too!)

This `down_revision` should be `7839740aa454`.

You can determine the correct `down_revision` using the `sep alembic heads` command and looking for the `(uber) (head)` revision:
```
$ sep alembic heads
7839740aa454 (uber) (head)
b4a97b074ef2 (panels, uber) (head)
9e3e15a7cf59 (panels, guests, uber) (head)
73b22ccbe472 (uber, attendee_tournaments) (head)
5dced3c6ef91 (uber, hotel) (head)
e68ef1dc43fc (uber, magprime) (head)
c31b1a94a27d (mits, uber) (head)
e4d09d36083d (uber, mivs) (head)
691be8fa880d (panels, uber, tabletop) (head)

```

Reply:
Fixed!  I'll see if I can take a look at that alembic bug this weekend, but that'll depend on getting through all the MITS and Hotel work, so fingers crossed!